qupath-extension-processing/src/main/java/qupath/imagej/plugins/ImageJMacroRunner.java

```diff
@@ -377,7 +377,7 @@ public String getLastResultsDescription() {
 	}
 
 
-	ParameterList getParameterList(final ImageData<BufferedImage> imageData) {
+	public ParameterList getParameterList( final ImageData<BufferedImage> imageData ) {
```

Comment:
This is so that we can access the parameters from the MacroRunner for the scripting part.